.claude/skills/shared/scripts/gh_helpers.sh

```
@@ -0,0 +1,147 @@
+#!/bin/bash
+# Shared GitHub CLI helper functions
+# Source this file to use the functions: source gh_helpers.sh
+
+set -e
+
+# Get repository owner and name from git remote
+get_repo_info() {
+    local remote_url=$(git config --get remote.origin.url)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)